### ISSUE TITLE
The Vita Amiga emulator does not see the kickstart correctly and does not start

### ISSUE THREAD

**Issue by Ted82** [rsn8887/uae4all2#37]
I have a PS Vita 3.60. I have a big problem with the Amiga emulator.After installing and throwing in the correct place the required kickstart (kick13, kick20, kick31)

ux0:/data/uae4all/kickstarts

The emulator starts, but the title screen shows: kickstart rom(s) not found. Please any button to exit.

I upload the files correctly, I have via USB access to the data directory on my computer. I do not know what to do. How do I solve this? I have no more ideas. Please help me and thank you in advance!

Translated with www.DeepL.com/Translator (free version)

**Comment by rsn8887** [comment]
Did you name them correctly? They have to be named kick13.rom not kick13. The ending .rom is important.

There are some bios files floating on the internet that don’t work with this Emulator. Try different ones. MD5 of verified working kickstart rom files, you can test yours at onlinemd5:

kick12.rom (256 kB): (MD5) 85AD74194E87C08904327DE1A9443B7A

kick13.rom (256 kB): (MD5) 82A21C1890CAE844B3DF741F2762D48D  

kick20.rom (512 kB): (MD5) FA4ACC75B49E880679FE02716AF24D71  

kick31.rom (512 kB): (MD5) 646773759326FBAC3B2311FD8C8793EE  

I hope this helps

**Comment by Ted82** [comment]
Yeas, every file have .rom ending. Is very strange, because on my friend PS Vita (3.65) this roms work without any problems. I dont know why. I try onlinemd5, thanks.

**Comment by Ted82** [comment]
I found correct kickstarts, verified, nothing changes. Still error - kickstarts not found. I dont know, maybe something wrong with data folder? Maybe the reason is, if i connect my vita to USB, system shows MTP USB device, no playstation vita? If i press select, i see new disk and i can copy files. And system doesnt see it (Amiga Emulator, PSP Adrenaline works great)

**Comment by rsn8887** [comment]
Maybe the file permissions are wrong or filenames are not lowercase? (KICK.ROM vs kick.rom). Try to copy the files to some other folder with usb, then copy them on the Vita  into the correct folder with Vitashell.

Alternatively, try FileZilla FTP client on Windows and Vitashell FTP Server on Vita to copy the files via ftp.

**Comment by Ted82** [comment]
Yes, me too, same files path. File permission? What do you mean? I never use Filezilla

**Comment by Ted82** [comment]
I think i found reason. In Uae4all tutorial i see info:

Second, on Vita, please make sure your ux0:/data folder does NOT have the FAT attributes "hidden" or "system" set. 

my data folder has hidden attribute, blank, i cant change it. In device manager in windows i change everything to show hidden folders etc, i can copy and paste everything. But this attribute still exist. How can i change it?

**Comment by rsn8887** [comment]
You can do it on the command prompt.

right click you memory stick and select "Command Prompt here"
enter `attrib –h –s data` press enter
